portal-impl/src/com/liferay/portlet/expando/service/impl/ExpandoValueLocalServiceImpl.java

```diff
@@ -1981,4 +1987,5 @@ public static List<ExpandoValueDeleteHandler> getService(String key) {
 					ExpandoValueDeleteHandler.class, "model.class.name");
 
 	}
+
```

Comment:
Your first commit breaks SF unnecessarily, and this commit resolves it.  Instead, squash this line change with the previous commit, so it never happened.